Backend/src/main/java/ca/mcgill/cooperator/service/NotificationService.java

```diff
@@ -112,6 +114,26 @@ public Notification getNotification(String title) {
         return ServiceUtils.toList(notificationRepository.findAll());
     }
 
+    /**
+     * set notification to seen
+     *
+     * @return notification seen
+     */
+    public Notification setSeen(Notification n) {
+    	StringBuilder error = new StringBuilder();
+    	if(n != null)
+    		n.setSeen(true);
+    	else {
+    		error.append("Notification cannot be null");
```

Comment:
Since there's only one error message you can just throw the exception here directly without needing the StringBuilder